frontend/src/components/CounselingCenterCard.js

```diff
@@ -38,21 +40,22 @@ const Wrapper = styled.button`
   border: none;
   border-radius: 12px;
   width: 100%;
-  display: flex;
-  align-items: center;
+  display: grid;
+  grid-template-columns: 1fr min-content;
+  grid-template-rows: 1fr min-content;
 
   img {
     width: 30px;
     height: 30px;
-    align-self: center;
+    justify-self: center;
+  }
+
+  > section {
```

Comment:
Which section does this target? ReviewStats? Wehen you expose a `className` prop on ReviewStats you can do like so with styled components
```
$(ReviewStasts){
  // some style rules
}
```

Reply:
Somehow that doesn't work for me 😅
What exactly do you mean by expose a className prop on ReviewStats? And where do I have to put it?